docs/commerce/shipping_management/shipping_management.md

```diff
@@ -0,0 +1,12 @@
+---
+description: Shipping management module handles shipping methods.
+edition: commerce
+---
+
+# Shipping management
+
+If the permissions assigned to your [user role](permissions_and_users.md) allow, the Shipping management module enables you to define and manage shipping methods of different types on the **Shipping methods** screen.
+
+![Shipping methods screen](shipping_methods_list.png "Shipping methods screen")
+
+The Shipping management module interacts with other packages of the system, so that shipment processing is cancelled automatically when the customer cancels their order.
```

Comment:
```suggestion
The Shipping management module interacts with other packages of the system, 
so that shipment processing is cancelled automatically when the customer cancels their order.
```